crates/compiler/src/download.rs

```diff
@@ -20,15 +21,14 @@ pub struct ShaderSource {
 }
 
 pub fn download_shaders(
-    root_crate_name: &str,
+    root_crate_location: &CrateLocation,
     root_crate_version: &Version,
```

Comment:
Version is a bit useless when targeting local, but we'd have to rework the dropdown choice to support local.